File: src/api/Coursework.Domain/Entities/Recruiter.cs
namespace Coursework.Domain.Entities
{
    public class Recruiter
    {
        public Recruiter()
        {
            Vacancies = new HashSet<Vacancy>();
        }

        public Guid GID { get; set; }

        public string FirstName { get; set; } = null!;

        public string LastName { get; set; } = null!;

        public string CompanyName { get; set; } = null!;

        public string PhoneNumber { get; set; } = null!;

        public string Biography { get; set; } = null!;

        public string Location { get; set; } = null!;

        public Guid UserGID { get; set; }

        public virtual User User { get; set; }

        public virtual ICollection<Vacancy> Vacancies { get; set; }

    }
}

File: src/frontend/src/app/recruiters/models/recruiter.ts
export class Recruiter {
  firstName: string | null = null;
  lastName: string | null = null;
  companyName: string | null = null;
  phoneNumber: string | null = null;
  biography: string | null = null;

  location: string | null = null;
  userGID: string | null = null;

  constructor(firstName: string,
              lastName: string,
              companyName: string,
              phoneNumber: string,
              biography: string,
              location: string,
              userGID: string) {
    this.firstName = firstName;
    this.lastName = lastName;
    this.companyName = companyName;
    this.phoneNumber = phoneNumber;
    this.biography = biography;
    this.location = location;
    this.userGID = userGID;
  }
}
